extension/CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
-# Change Log
-All notable changes to the "vscode-java-test" extension will be documented in this file.
-
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
-
-## [Unreleased]
-- Initial release
\ No newline at end of file
+# Change Log
+All notable changes to the "vscode-java-test" extension will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## 0.1.0 - 2017-12-01
+### Added
+- Support JUnit (v4.8.0+) test cases
+- Run test cases
+- Debug test cases
+- View test logs
+
+## 0.2.0 - 2017-12-21

```

Comment:
Should be in the reverse order of the timeline, which the latest should put on the top.